hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/qjournal/server/TestJournalNode.java

```diff
@@ -281,7 +300,6 @@ public void testJournal() throws Exception {
 
   }
   
-  
   @Test(timeout=100000)
```

Comment:
Could you please readd the line here?

Reply:
There were tow empty lines so I remove one empty line.